Controller/ExceptionController.php

```diff
@@ -26,8 +26,10 @@
 /**
  * Custom ExceptionController that uses the view layer and supports HTTP response status code mapping.
  */
-class ExceptionController extends ContainerAware
+class ExceptionController
```

Comment:
you need to implement `ContainerAwareInterface`. Otherwise I don't thikn, the container will be set at all.